app/src/main/java/com/mobilecoronatracker/data/persistence/dao/AccumulatedDataDao.kt

```diff
@@ -30,12 +30,14 @@ abstract class AccumulatedDataDao : BaseDao<AccumulatedData> {
         SELECT * 
         FROM accumulated_data 
         WHERE entry_date>=:timestampStart AND entry_date<=:timestampEnd
+        ORDER By entry_date
```

Comment:
Minor: `By` should be `BY` to make consistent use of the SQL statements as UPPERCASE words

Reply:
fixed